tests/test_product_requirements_text_QCI97.py

```diff
@@ -0,0 +1,77 @@
+# QCI-97
+
+# In the "Select products to deploy screen, there is a text box to the right
+# of the products list.   It as you select various products will contain text
+# that explains what is required by each product selected.   Be sure the text
+# is correct for the products selected.
+
+def test_product_requirements_text(new_deployment_pg):
+    general_expected_text = (
+        "General\nRed Hat Customer Portal username and password. This account"
+        " must be an Organization Administrator within the Portal\nSufficient"
+        " subscriptions to complete this deployment\nHost machines are started"
+        " and discoverable\nNetworks and subsets are set up and available"
+    )
+    rhv_expected_text = "\n".join([
```

Comment:
We can create a new pytest command line argument to specify the file to read for QCI expected text. So you don't have to store the file name in the variables.yaml file. The new command line arg can be included in pytest.ini and overriden manually on the command line

example: pytest <some test case> --variables variables.yaml --qci-expected-text <path to expected text filename>